src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -253,29 +254,51 @@ public StartEventResponse startEvent(
         CcdAuthenticator authenticator,
         String jurisdiction,
         String caseTypeId,
-        String caseRef,
         String eventTypeId
     ) {
-        return caseRef == null
-            ? feignCcdApi.startForCaseworker(
-            authenticator.getUserToken(),
-            authenticator.getServiceToken(),
-            authenticator.getUserDetails().getId(),
-            jurisdiction,
-            caseTypeId,
-            eventTypeId
-        )
-            : feignCcdApi.startEventForCaseWorker(
+        return feignCcdApi.startForCaseworker(
             authenticator.getUserToken(),
             authenticator.getServiceToken(),
             authenticator.getUserDetails().getId(),
             jurisdiction,
             caseTypeId,
-            caseRef,
             eventTypeId
         );
     }
 
+    public StartEventResponse startEventForExistingCase(
+        CcdAuthenticator authenticator,
+        String jurisdiction,
+        String caseTypeId,
+        String caseRef,
+        String eventTypeId
+    ) {
+        try {
+            return feignCcdApi.startEventForCaseWorker(
+                authenticator.getUserToken(),
+                authenticator.getServiceToken(),
+                authenticator.getUserDetails().getId(),
+                jurisdiction,
+                caseTypeId,
+                caseRef,
+                eventTypeId
+            );
+        } catch (FeignException e) {
+            if (e.status() == 404) {
+                throw new UnableToAttachDocumentsException(
```

Comment:
the method is called `startEventForExistingCase` and it can have nothing to do with attaching documents

Reply:
This is called from https://github.com/hmcts/bulk-scan-orchestrator/blob/28d55bfd9c583dc96a71f0048c07359e60f073f6/src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/AttachDocsToSupplementaryEvidence.java#L54

If the CCD event id (`attachScannedDocs`) is not configured for the case type, CCD will return 404 response.

Reply:
Doesn't matter where it's called from **right now**.
The method name has nothing to do with attaching docs.
You can rename it.

Reply:
renamed the method names for both start and submit events